pkg/controller/job/job_controller.go

```diff
@@ -1951,6 +1945,25 @@ func countReadyPods(pods []*v1.Pod) int32 {
 	return cnt
 }
 
+// terminatingFieldValue returns the value to be set as the "terminating" field in status
+func terminatingFieldValue(jobCtx *syncJobCtx) *int32 {
```

Comment:
Would it make sense to define it as a receiver method in `syncJobCtx`?

```
func (c *syncJobCtx) getTerminatingCount() *int32
```

Reply:
Do we need this function? Wouldn't it be better to inline it, since it is only used in a single place to reduce the indirection?  

The function name does not help either in the current state and one might think of it as a generic getter of the value.

Reply:
I agree, the function is likely to be always called from just one place. I will update.

Reply:
Done. PTAL